README.md

```diff
@@ -104,7 +104,7 @@ Here are some things to try.
 
 Once installed as described above, type
 
-    $ source venv/3.7/bin/activate
+    $ source venv/3.10/bin/activate
```

Comment:
It says "once installed as described above" but this venv and the directory layout `venv/<version>` is created in the makefile. The makefile step `make devready` is below in `Installing for development`. Maybe before the pip install above we should add a step about creating a virtual environment with `python3.10 -m venv venv/3.10`, or add it to this section. 

Should we also put a note that Python 3.10 must be installed? The makefile won't run otherwise.

(I have mixed opinions on tying the makefile to a python version)

Reply:
@theferrit32 good catch. We should make an issue about reorganizing the readme. I know that people often miss the submodule step during development

I do like the note about making sure python 3.10 is installed